### DISCUSSION TITLE
Add word wrap to "followed" cards

### DISCUSSION THREAD

**Discussion by aaossa** [community/community#36671] [Other Feature Feedback, Questions, & Ideas]
Hi, this is more like a "bug report" I think, but here we go. I've noticed that when a user followed another user or an organization, the text description of the profile is not properly broken into multiple lines. Of course, the profiles include the proper use of spaces.

![image](https://user-images.githubusercontent.com/10425834/196720566-328239a1-d1d1-40ee-ab16-1818a342014e.png)

![image](https://user-images.githubusercontent.com/10425834/196720102-ae81428c-8f08-4c82-9b2d-e45d08d86854.png)

On starred repositories, line breaks are marked adding a `-` character, which is good.

![image](https://user-images.githubusercontent.com/10425834/196721806-3f1768d2-9935-45a5-9aba-4b15d85e086a.png)

But I think that moving the word to the next line would look better.

**TLDR:**

Current behavior on profiles:

```
A research group specialized on Theoretical Computer Science whic
h includes Database Theory, Algorithmics, Computational Complexit
y, Logic and many more.
```

Current behavior on repos:
```
A research group specialized on Theoretical Computer Science whic-
h includes Database Theory, Algorithmics, Computational Complexit-
y, Logic and many more.
```

Expected behavior:

```
A research group specialized on Theoretical Computer Science 
which includes Database Theory, Algorithmics, Computational 
Complexity, Logic and many more.
```

**Comment by ossanna16** [comment]
Hey there @aaossa and thank you for reporting this bug! I'm glad you found our community 🙂
